packages/flutter_tools/lib/src/commands/assemble.dart

```diff
@@ -295,7 +295,7 @@ class AssembleCommand extends FlutterCommand {
       decodedDefines = decodeDartDefines(environment.defines, kDartDefines);
     } on FormatException {
       throwToolExit(
-        'Error parsing assemble command: your generated configuration may be out of date. '
+        'Error parsing assemble command: your --DartDefines are not base64 encoded or your generated configuration may be out of date. '
```

Comment:
chore:

```suggestion
        'Error parsing assemble command: either your --DartDefines options aren't base64 encoded or your generated configuration may be out of date. '
```

Reply:
Hi Alberto, thanks for your suggestions. I adjusted my PR accordingly with the last commit.